### ISSUE TITLE
Breaking local development build

### ISSUE THREAD

**Issue by josteph** [shadowwalker/next-pwa#97]
# Summary

Currently unable to use `next-pwa` with `next dev` command, caused by this line
https://github.com/shadowwalker/next-pwa/blob/9a95117b78e147d042f2fe9eb280d0517a84ad20/index.js#L235

# How To Reproduce

1. Install `next-pwa@latest`
2. Run `yarn dev`
3. Build error:
```
error - Couldn't find configuration for either precaching or runtime caching. Please ensure that the various glob options are set to match one or more files, and/or configure the runtimeCaching option.
```

# Expected Behaviors
Should not break in development environment

# Screenshots
![image](https://user-images.githubusercontent.com/13672022/90976691-2d25cc00-e569-11ea-9167-10d882dcb588.png)

**Comment by shadowwalker** [comment]
Fixed with version `3.1.3`. Please verify.

**Comment by josteph** [comment]
Cool, thanks for the fast response!
